src/main/java/com/squareup/kotlinpoet/FunSpec.kt

```diff
@@ -55,12 +55,8 @@ class FunSpec private constructor(builder: Builder) {
     }
   }
 
-  internal fun parameter(name: String): ParameterSpec? {
-    for (parameter in parameters) {
-      if (parameter.name == name) return parameter
-    }
-    return null
-  }
+  internal fun parameter(name: String): ParameterSpec? =
```

Comment:
Thought about keeping it to make it more clear that it's nullable, but it's probably clear anyway